docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -581,3 +598,85 @@ If processing of any rule fails, a partial error outputs the ID and/or name of t
 --------------------------------------------------
 
 <1> dev:[] `execution_summary` is under development and its schema may change.
+
+*Example 3, Dry run*
+
+The following request validates `add_index_patterns` action to the rules, which IDs included in the payload.
+Because, it's in dry ryn mode: no changes will be saved for these rules.
+
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/rules/_bulk_action?dry_run=true
+{
+    "action": "edit",
+    "edit": [
+        {
+            "value": [
+                "test-*"
+            ],
+            "type": "add_index_patterns"
+        }
+    ],
+    "ids": ["81aa0480-06af-11ed-94fb-dd1a0597d8d2", "dc015d10-0831-11ed-ac8b-05a222bd8d4a", "de8f5af0-0831-11ed-ac8b-05a222bd8d4a"]
+}
+--------------------------------------------------
+
+[discrete]
+===== Response code
+
+`500`::
+    Indicates partial bulk action failure.
+
+[discrete]
+===== Response payload
+
+Response payload returns 2 failed rules and 1 successful. In `attributes.errors` section of response, can be found error messages with list of failed rules(id and name).
+Contrary to the above example, that returns updated rule objects in `attributes.results`, in case of dry run mode, no rules objects will be returned, as rules are not getting saved in Elasticsearch.
```

Comment:
```suggestion
Notice that in the `attributes.results` all the arrays are empty. In the dry run mode, the endpoint doesn't return any `updated`, `created`, or `deleted` rules in the results, because no updates are being applied and saved to Elasticsearch.
```

Reply:
```suggestion
Notice that there are no arrays in `attributes.results`. In the dry run mode, rule updates are not applied and saved to {es}, which means the endpoint won't return results for rules that have been `updated`, `created`, or `deleted`.
```